### DISCUSSION TITLE
Is Safari Browser Supported

### DISCUSSION THREAD

**Discussion by CarrieEBeaton** [katspaugh/wavesurfer.js#3383] [Q&A] [answered]
The wavesurfer won't render on Safari.  Is this browser supported and do I have to do something specific to get it to render on that browser?  I have a wavesurfer with the regions and timeline plugins added.  Thanks in advance for your help.

**Comment by katspaugh** [comment, marked as answer]
Yes, it's supported as you can see on https://wavesurfer.xyz.
Some audio files might not decode on Safari. You can check this compatibility table: https://caniuse.com/?search=audio%20format

**Reply by CarrieEBeaton** [reply]
I was using a .wav file and I found that making this change was the fix - set the following backend: 'WebAudio' (I had MediaElement as the option here beforehand).
